Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx

```diff
@@ -165,12 +167,8 @@ ComparisonImageFilter<TInputImage, TOutputImage>::ThreadedGenerateData(const Out
         InputPixelType t = valid.Get();
 
         //  Assume a good match - so test center pixel first, for speed
-        RealType difference = static_cast<RealType>(t) - test.GetCenterPixel();
-        if (NumericTraits<RealType>::IsNegative(difference))
-        {
-          difference = -difference;
-        }
-        auto minimumDifference = static_cast<OutputPixelType>(difference);
+        RealType difference = std::abs(static_cast<RealType>(t) - test.GetCenterPixel());
+        auto     minimumDifference = static_cast<OutputPixelType>(difference);
```

Comment:
@thewtex Regarding our discussion at [discourse](https://discourse.itk.org/t/how-to-assert-that-images-are-equal-in-python-unit-test/7302) So yes, `ComparisonImageFilter` _does_ take the absolute value! (This pull request just aims to make that clearer. It does not change the behavior, as far as I can see.)

Reply:
@N-Dekker great!